src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/errors/UsersManageError.kt

```diff
@@ -0,0 +1,7 @@
+package org.vaccineimpact.orderlyweb.errors
+
+import org.vaccineimpact.orderlyweb.models.ErrorInfo
+
+class UsersManageError(message: String): OrderlyWebError(400, listOf(
```

Comment:
This class feels a bit over-specific; `InvalidOperationError` might be more re-usable?